modules/UI.js

```diff
@@ -0,0 +1,54 @@
+const booksContainer = document.querySelector('.book-list-container');
+
+export default class UI {
+  // Get Books from Local Storage and Updates the UI
+  static updateBooks(booksContainer) {
+    const books = JSON.parse(localStorage.getItem('books'));
+    booksContainer.innerHTML = '';
+    if (books) books.forEach((book) => this.addBook(book));
+  }
+
+  // Creates the UI's Book
+  static addBook(book) {
```

Comment:
Kindly use the ES6 arrow function syntax instead of the function declaration where ever possible. This is one of the requirements for this milestone. ⬇️
![image](https://user-images.githubusercontent.com/33268927/213628825-94c7fa25-dd48-4f37-a071-24e1adbb809e.png)